src/localizers/globalize.js

```diff
@@ -37,13 +37,28 @@ export let formats = {
 export default function(globalize) {
   let locale = culture => culture ? globalize(culture) : globalize;
 
+  // return the first day of the week from the locale data. Defaults to 'world'
+  // territory if no territory is derivable from CLDR.
+  // Failing to use CLDR supplemental (not loaded?), revert to the original
+  // method of getting first day of week.
   function firstOfWeek(culture) {
-    let date = new Date();
-    //cldr-data doesn't seem to be zero based
-    let localeDay = Math.max(
-      parseInt(locale(culture).formatDate(date, { raw: 'e' }), 10) - 1, 0)
-
-    return Math.abs(date.getDay() - localeDay)
+    try {
+        const days = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```

Comment:
Those are constants used in the cldr to denote a day. They will be the same for all locales, as far as I know. Those are just used to determine the numeric value of the day. The same method is used inside Globalize to get first day of the week (though their method is not exposed outside). The supplemental package of CLDR contains one block of all the territories (by country code) and their first day of the week using those markers.